locale/English.json

```diff
@@ -65,7 +65,8 @@
       "langChange": "Select your language",
       "defaultScreenPick": "Activity as default screen",
       "recordActivity": "Record Activity",
-      "language": "Language"
+      "language": "Language",
+      "inActivityAlert": "Switch if no Scan in 30s"
```

Comment:
Nope! Switch is the action (by toggling the button)... Lets try to reword it a bit.

"Scanner Timeout" is much apt for this!

Reply:
Also, notice that this is not related to 'activity' at all, it is related to Scanner :-)